api/lib/cat/answer.js

```diff
@@ -0,0 +1,21 @@
+class Answer {
+  constructor(challenge, result) {
+    this.challenge = challenge;
+    this.result = result;
+  }
+
+  get binaryOutcome() {
+    if(this.result === 'ok') {
```

Comment:
Ou simplement une ternaire, ce serait pas gênant ici.
`return if(this.result === 'ok') ? 1 : 0;`